src/components/PasswordReset/index.js

```diff
@@ -0,0 +1,92 @@
+import React, { Component } from 'react';
+import './index.css';
+import { connect } from 'react-redux';
+import { bindActionCreators } from '../../../../../../Library/Caches/typescript/3.1/node_modules/redux';
```

Comment:
this import is too long.  You can import from redux directly 'import { bindActionCreators } from 'redux';